docs/reference/modules/network.asciidoc

```diff
@@ -35,11 +34,23 @@ in the following table:
 
 |`_local:ipv6_` |Will be resolved to loopback IPv6 addresses
 
-|`_non_loopback_` |Addresses of the first non loopback interface
+|`_site_` |Will be resolved to site-local addresses ("private network")
 
-|`_non_loopback:ipv4_` |IPv4 addresses of the first non loopback interface
+|`_site:ipv4_` |Will be resolved to site-local IPv4 addresses
 
-|`_non_loopback:ipv6_` |IPv6 addresses of the first non loopback interface
+|`_site:ipv6_` |Will be resolved to site-local IPv6 addresses
+
+|`_global_` |Will be resolved to globally-scoped addresses ("publicly reachable")
```

Comment:
just because the address has global scope does not its unprotected. The idea here is to just be explicit and consistent.